kit/popups/src/Sheets/UICardSheet.tsx

```diff
@@ -19,29 +32,47 @@ function getCardSheetBottomInset(bottomInset: number, keyboardHeight: number) {
     return Math.max(CARD_SHEET_DEFAULT_BOTTOM_INSET, bottomInset);
 }
 
-export function UICardSheet({ children, style, ...rest }: UICardSheetProps) {
+export function UICardSheet({
+    children,
+    style,
+    hasHeader = true,
+    headerOptions,
+    ...rest
+}: UICardSheetProps) {
+    const theme = useTheme();
     const { visible, forId } = rest;
+    const styles = useStyles(theme);
     return (
         <UISheet.Container visible={visible} forId={forId}>
             <UISheet.KeyboardAware getBottomInset={getCardSheetBottomInset}>
                 <UISheet.IntrinsicSize>
-                    <UISheet.Content {...rest} style={styles.card}>
-                        <View style={[style, styles.cardInner]}>{children}</View>
+                    <UISheet.Content {...rest} style={styles.sheet}>
+                        <View style={styles.card}>
+                            {hasHeader ? <UIDialogBar hasPuller {...headerOptions} /> : null}
+                            <View style={[styles.cardContent, style]}>{children}</View>
```

Comment:
I'm not sure that a `style` should passed here, as for now we apply all styles ourselfes, maybe the `style` isn't needed at all

Reply:
We need this so that user can style the container for the content (paddings, flex direction, alignment of elements)

Reply:
I don't why GH put my answer there https://github.com/tonlabs/UIKit/pull/1491#discussion_r799246851 :)

Reply:
OK, I agree

Reply:
OK, I agree.